plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/core/AwsSdkClient.kt

```diff
@@ -32,8 +37,29 @@ class AwsSdkClient : SdkClientProvider, Disposable {
         ValidateCorrectThreadClient(httpClientBuilder.build())
     }
 
+    private val sdkAsyncHttpClient: SdkAsyncHttpClient by lazy {
+        LOG.info { "Create new async Netty NIO client" }
+        val proxy = CommonProxy.getInstance().select(java.net.URI(CodeWhispererConstants.Config.CODEWHISPERER_ENDPOINT)).first()
+        val address = proxy.address()
+        val clientBuilder = NettyNioAsyncHttpClient.builder()
+
+
+
+        if (address is java.net.InetSocketAddress) {
+            val trustManagers = arrayOf(CertificateManager.getInstance().trustManager)
+            val x509TrustManagers = trustManagers.map { it as X509TrustManager }.toTypedArray()
```

Comment:
you can collapse down to `arrayOf<TrustManager>(CertificateManager.getInstance().trustManager)`

Reply:
Done